chat-plugins/wifi.js

```diff
@@ -437,6 +437,17 @@ var commands = {
 	}
 };
 
+var giveawayRoom = Rooms.get('wifi');
+if (giveawayRoom) {
+	if (giveawayRoom.plugin) {
+		giveaway = giveawayRoom.plugin.giveaways;
```

Comment:
Hmm, I just literally followed a similar style to the jeopardy plugin and wrote it just before exports.commands in the file